azuredeploy.bicep

```diff
@@ -105,21 +107,235 @@ param storageType string = 'Standard_LRS'
 @description('Name of the container used to store backing files in the new storage account. This container is created automatically during deployment.')
 param storageContainerName string = 'redcap'
 
+param vmAdminUserName string
```

Comment:
I don't necessarily think a username is a secret, but in most of our samples, the username is a secret param too.